src/index.js

```diff
@@ -28,4 +28,7 @@ function fetchPuzzles() {
   fetch("https://fe-apps.herokuapp.com/api/v1/gametime/1903/wheel-of-fortune/data")
     .then(response => response.json())
     .then(data => data.data.puzzles.one_word_answers.puzzle_bank.forEach(puzzle => console.log(puzzle)))
+
+    // The .then() statement below does not work as written, but we need to do something like that I think. I think it doesn't work because of fetch's async rules. 
+    // .then(data => data.data.puzzles.one_word_answers.puzzle_bank.forEach(puzzle => puzzleBank.unshift(puzzle)))
```

Comment:
this is where we need to trouble shoot. .unshift() does not work within the fetch. It works in the console, but not when I run it like this. I think the browser is smarter than the JS engine and works around the async vs single-threaded issue.